src/Microsoft.Sbom.Api/Config/SbomToolCmdRunner.cs

```diff
@@ -131,5 +139,22 @@ public async Task Generate(GenerationArgs generationArgs)
                 IsFailed = true;
             }
         }
+
+        /// <summary>
+        /// Prints the version of the tool.
+        /// </summary>
+        [ArgActionMethod]
+        [ArgDescription("Displays information about the version of the tool being used.")]
+        public void Info()
+        {
+            if (!string.IsNullOrEmpty(Version))
+            {
+                Console.WriteLine($"SBOM Tool Version: {Version}");
```

Comment:
nit: I would say just `Console.WriteLine(Version);` should be enough. It would make it easier for automatic tools to parse version